project-saarah/scenes/add-product-item/AddProductItemModels.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  AddProductItemModels.swift
+//  project-saarah
+//
+//  Created by Guilherme Colombini on 17/10/19.
+//  Copyright (c) 2019 Bruno Rocha. All rights reserved.
+//
+
+import Foundation
+
+enum AddProductItem {
+	struct AddItemForm {
+		let quantity: Double
+		let price: Double
+		let expirationDate: String
+	}
+	// MARK: Use cases
+	enum AddItem {
+		struct Request {
+			let addItemForm: AddItemForm
+		}
+		struct Response {
+			let apiMessage: String
```

Comment:
I don't know if you put this 'apiMessage' just for testing, but avoid to use terms like 'api', 'coredata', or any other that refers  about the way of how its was implemented. Just 'message' its good enough.